App/CourseApp/Views/SwiftUI/SwipingView.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  SwipingView.swift
+//  CourseApp
+//
+//  Created by Marcel Mravec on 17.05.2024.
+//
+
+import os
+import SwiftUI
+
+struct SwipingView: View {
+    private let dataProvider = MockDataProvider()
+    @State private var flipped = false
```

Comment:
Three flags is still file. Having even more maybe some wrapper configuration struct would be beneficial.